recipes/astrometry.net/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "astrometry.net" %}
+{% set version = "0.78" %}
+
+package:
+  name: {{ name|lower}}
+  version: {{ version }}
+
+source:
+  fn: {{name}}-{{ version }}.zip
+  url: https://github.com/dstndstn/astrometry.net/archive/{{ version }}.zip
+  sha256: 85480b9636757c89ef5ccc7aac557409ac2ab377eb6c6eca862b16cdd4faefd9
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - swig
+    - {{ compiler('c') }}
+    - git
+    - make
+    - {{ cdt('libx11-devel') }}  # [linux]
+    - {{ cdt('libxau-devel') }}  # [linux]
+    - {{ cdt('libxext-devel') }}  # [linux]
+    - {{ cdt('libxrender-devel') }}  # [linux]
+    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
```

Comment:
Aren't these cdt's host deps?  And I think a lot of these CDTs have real packages now

Reply:
For these system libraries, it is better to use anaconda packages than cdt? I don't know exactly when to use what.

Regarding proper anaconda dependecies, conda-forge has all these libraries, except `xorg-X11-proto-devel`.